### PR TITLE
[R4] MedicationRequest Add Sweden HardCoded extensions

### PR DESCRIPTION
**Pull request by HP082979**
Added Hardcoded Extension to MedicationRequest

Description
----
Screen Shot 2021-01-25 at 3 58 51 PM
Screen Shot 2021-01-25 at 4 00 28 PM
Screen Shot 2021-01-25 at 4 02 09 PM

PR Checklist
----
- [X] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

File: content/millennium/r4/medications/medication-request.md [hunk 1]
```diff
@@ -59,6 +59,20 @@ The following fields are returned if valued:
 
 <%= terminology_table(:medication_request_contained_medication, :r4) %>
 
+## Extensions
+
+* [NLLPrescriptionFormat](http://electronichealth.se/fhir/StructureDefinition/NLLPrescriptionFormat)
```
Comment (fpivonka) [inline content/millennium/r4/medications/medication-request.md:64]: I think these need to link out to the Swedish Specification. https://simplifier.net/guide/swedishnationalmedicationlist/MedicationRequest. If we look at examples of extensions on the Patient Resource the link on the extension name links to the specification.

File: content/millennium/r4/medications/medication-request.md [hunk 2]
```diff
@@ -59,6 +59,20 @@ The following fields are returned if valued:
...
+* [NLLRegistrationBasis](http://electronichealth.se/fhir/StructureDefinition/NLLRegistrationBasis)
+* [NLLDosePackaging](http://electronichealth.se/fhir/StructureDefinition/NLLDosePackaging)
+
+All URLs for hardcoded extensions are defined as `http://electronichealth.se/fhir/StructureDefinition/{id}`
```
Comment (fpivonka) [inline content/millennium/r4/medications/medication-request.md:68]: I don't think we want 'hardcoded' listed here. Maybe it should be something like "All URLs for Swedish extensions are defined as `http://electronichealth.se/fhir/StructureDefinition/{id}`

**Review by fpivonka — CHANGES_REQUESTED** [review]

**Review by praneethk2094 — APPROVED** [review]
+1 after Frans proposals.

File: content/millennium/r4/medications/medication-request.md (hunk 2 above)
Comment (HP082979) [reply content/millennium/r4/medications/medication-request.md]: Fix for this:
https://github.com/cerner/fhir.cerner.com/commit/a92347cb1bc1b50ba45b882417bda299dfc99c6b

File: content/millennium/r4/medications/medication-request.md (hunk 1 above)
Comment (HP082979) [reply content/millennium/r4/medications/medication-request.md]: Fix for this: 
https://github.com/cerner/fhir.cerner.com/commit/a92347cb1bc1b50ba45b882417bda299dfc99c6b

**Review by fpivonka — APPROVED** [review]

**Review by ab082977 — APPROVED** [review]

**Comment by MaxPhilips** [comment]
post-merge validation
Screen Shot 2021-02-01 at 4 40 34 PM

**Pull request merged**